requirements.txt

```diff
@@ -23,3 +23,4 @@ numba
 emoji
```

Comment:
make sure to add versions for these (see output of `pip3 freeze`). Otherwise easy to have future versions break changes

Reply:
resolved but added extra requirements. Has been resolved on updated comment